cookiecutter-frontend-app/{{cookiecutter.repo_name}}/.env.test

```diff
@@ -1,5 +1,5 @@
 ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
-BASE_URL='localhost:1995'
+BASE_URL='localhost:{{cookiecutter.port}}'
```

Comment:
This had fallen out of sync from `.env.development`, so searching for `8080` wouldn't catch this one.